<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="projectLabel">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="projectList"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="libraryRadio">
        <property name="text">
         <string>Library</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="filmRadio">
        <property name="text">
         <string>Film</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="printRadio">
        <property name="text">
         <string>Print</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="familyLabel">
        <property name="text">
         <string>Family</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="familyInput"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>OR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="familyList">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="assetLabel">
        <property name="text">
         <string>Asset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="assetInput"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>OR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="assetList">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="deptLabel">
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Dept  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="deptList">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QRadioButton" name="createOpenRadio">
         <property name="text">
          <string>Create and Open</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="createRadio">
         <property name="text">
          <string>Create Only</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="saveRadio">
         <property name="text">
          <string>Save As</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="createButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
   <zorder></zorder>
   <zorder></zorder>
   <zorder></zorder>
   <zorder></zorder>
   <zorder></zorder>
   <zorder>groupBox_2</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>createButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>363</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
